src/structures/Webhook.js

```diff
@@ -103,6 +103,7 @@ class Webhook {
    * @property {FileOptions[]|BufferResolvable[]|MessageAttachment[]} [files] See {@link BaseMessageOptions#files}
    * @property {MessageMentionOptions} [allowedMentions] See {@link BaseMessageOptions#allowedMentions}
    * @property {MessageActionRow[]|MessageActionRowOptions[]} [components]
+   * @property {MessageAttachment[]} [attachments]
    * Action rows containing interactive components for the message (buttons, select menus)
```

Comment:
By inserting the added line after `components`, the description for `components` is now attached to `attachments`. They should be swapped around, and ideally, a description should be added to `attachments`. Maybe akin to what the other options are?

Reply:
Will do that, my bad